src/Test.FunctionalTests.BinSkim.Driver/UpdateBaselines.ps1

```diff
@@ -33,8 +33,8 @@ function Build-Baselines($sourceExtension)
 
         # Actually run the tool
         Remove-Item $outputTemp -ErrorAction SilentlyContinue
-        Write-Host "$utility analyze "$input" --output "$outputTemp" --kind "Fail;Pass" --level "Error;Warning;Note" --insert Hashes --remove NondeterministicProperties --config default --quiet true"
-        &           $utility analyze "$input" --output "$outputTemp" --kind "Fail;Pass" --level "Error;Warning;Note" --insert Hashes --remove NondeterministicProperties --config default --quiet true
+        Write-Host "$utility analyze "$input" --output "$outputTemp" --kind "Fail`;Pass" --level "Error`;Warning`;Note" --insert Hashes --remove NondeterministicProperties --config default --quiet true"
```

Comment:
>Write [](http://example.com/codeflow?start=8&length=5)

Our update command wasn't working for me without escaping the semicolons in these lines.